ScnSoft-Education/LeetCodeTraining.Tests/3SumTest.cs

```diff
@@ -0,0 +1,55 @@
+﻿using LeetCodeTraining.ThreeSum;
+
+namespace LeetCodeTraining.Tests
+{
+    public class ThreeSumSolutionTest
+    {
+        [Theory]
+        [MemberData(nameof(Data))]
+        public void ThreeSumTest(int[] nums, IList<IList<int>> expectedResult)
+        {
+            // Act
+            var result = ThreeSumSolution.ThreeSum(nums);
+
+            // Assert
+            Assert.Equal(expectedResult.Count, result.Count);
```

Comment:
It seems the assertion below can be enough for testing:
`Assert.Equal(expectedResult, result);`